src/router/appRoutes.ts

```diff
@@ -0,0 +1,14 @@
+const List = () => import(/* webpackChunkName: "about" */ "../views/Cards.vue");
```

Comment:
Its important to change `webpackChunkName` to another name like `cards`, because on the release dist folder the webpack will split all on 'about'.